.travis/setup_linux_environment.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ev
+
+# Config & Install
+gem install bundler --version= 2.2.19
```

Comment:
There is a space here.
Also you could use this trick to make it work forever

```suggestion
gem install bundler --version=$(grep -A 1 "BUNDLED WITH" Gemfile.lock | tr -d ' '| tail -n 1)
```